inttest/dualstack/dualstack_test.go

```diff
@@ -191,6 +199,17 @@ func (s *DualstackSuite) SetupSuite() {
 	s.client = client
 }
 
+func (s *DualstackSuite) getIPv6Address(nodeName string) string {
+	ssh, err := s.SSH(s.Context(), nodeName)
+	s.Require().NoError(err)
+	defer ssh.Disconnect()
+
+	ipAddress, err := ssh.ExecWithOutput(s.Context(), "ip -6 -oneline addr show eth0 | awk '{print $4}' | grep -v '^fe80' | cut -d/ -f1")
```

Comment:
adding a `| head -1` is probably more reliable, but I think we'd prefer this to break if there was a change in the `ip` command output.